help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+|--------------------------------------------------------------------------------------------------------------------------------	|----------------------	|--------------------------	|------------------------------------------------------------------	|-----------	|----------------------	|--------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------------------------------------------------------------	|
+| text                                                                                                                           	| in full if available 	| same                     	| lastname, firstname m.                                           	| **%A**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S. and Accomazzi, A.<br>-Kurtz, Michael J. and Accomazzi, Alberto 	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, Michael J. and Accomazzi, Alberto 	|
+| symbol                                                                                                                         	| in full if available 	| same                     	| lastname, firstname m.                                           	| **%a**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., & Accomazzi, A.<br>-Kurtz, Michael J. & Accomazzi, Alberto    	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz, Michael J. & Accomazzi, Alberto             	|
+| none                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%G**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., Accomazzi, A.<br>-Kurtz, M. J., Accomazzi, A.                 	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz, M. J., Accomazzi, A.                        	|
+| none                                                                                                                           	| initials             	| same                     	| lastname f. m.note no comma after lastname                       	| **%g**    	| , et al.             	|  -Henneken E. A., Muench G., Holm Nielsen L., Blanco-Cuaresma S., Accomazzi A.<br>-Kurtz M. J., Accomazzi A.                        	|  -Henneken E. A., Muench G., et al.<br>-Kurtz M. J., Accomazzi A.                            	|
```

Comment:
* In the format column, add a space or a `<br>` between `lastname f. m.` and `note`
* Capitalize the first letter in `Note`
* Same comments for `%e` as well

Reply:
Did not capitalize `note` since we have to decide if all of the values in the columns should be capitalized, for example for `%H` we have `first author only if number of authors > 2 otherwise both authors`, if we capitalize `note` we should capitalize `first`.

Reply:
Yeah, I think with the `<br>` separating the note from the rest, it looks fine without being capitalized. I just wanted the "note" section to have some kind of visual separation from the first part of the cell